cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -418,6 +422,28 @@ def setUp(self):
         # Create a second sequential just (testing children of children)
         self.create_xblock(parent_usage_key=self.chapter_usage_key, category='sequential2')
 
+    def test_discussion_duplication(self):
+        """
+        Tests that duplicated discussion component have unique 'discussion_id',
+        if 'discussion_id' is present in xml.
+        """
+        store = modulestore()
+        # Importing course from xml
+        courses = import_course_from_xml(
```

Comment:
@adampalay We can create new course using `CourseFactory`, but the scenario was such 

1. Import a course with `discussion_id` assigned in xml
2. Duplicate discussion & `discussion_id` should not be duplicated.
So in this case, the course we are importing already has `discussion_id` which we defined in the xml & we are testing if it does not have same ids once duplicated. 

Let me know if it is not clear.